components/AppServices/src/CommunityToolkit.AppServices.targets

```diff
@@ -1,5 +1,10 @@
 <Project>
 
+  <!-- Allow the source generators to detect whether an app on modern .NET is UWP or not -->
+  <ItemGroup>
+    <CompilerVisibleProperty Include="UseUwpTools" />
```

Comment:
If this is needed for any modern .NET on UWP project that uses source generators, we should add this to our tooling as well. We have SampleGen source generators that'll need to work under modern .NET for UWP.

Reply:
No, this is not general. We only need this here in these .targets, which are not evaluated by the project during build, but rather they're packaged in the NuGet and imported by _consuming_ projects. They're needed so the generator can see whether the project being built is UWP or not. But it's not needed otherwise. This should just remain here.